extensions/parquet/base/src/main/java/io/deephaven/parquet/base/BulkWriter.java

```diff
@@ -97,4 +97,12 @@ int writeBulkVector(@NotNull final BUFFER_TYPE bulkValues,
      */
     ByteBuffer getByteBufferView() throws IOException;
 
+    /**
+     * This function is used to provide a specialized definition of NULL for this writer. For example, if setNull(65535)
+     * is called, then 65535 will be written as NULL by this writer
+     *
+     * * @param nullValue The value which will be interpreted as NULL for this writer
+     */
+    void setNull(int nullValue);
```

Comment:
See proposed changes to naming and JavaDoc in `PlainIntChunkedWriter`. This should get most of those changes, possibly absent the specific commentary about other types.

If we don't make changes to just figure out `nullValue` in the constructor, I think we should consider making this take `Object`, so we can blindly call it for every type of `ColumnWriter`. The non-int implementations can just verify that they got the expected value. The int implementation can unbox the value to an `int`, probably by casting to `Number` and calling `intValue()`.